src/smt/env.h

```diff
@@ -61,7 +61,7 @@ class Env
   /**
    * Construct an Env with the given node manager.
    */
-  Env(NodeManager* nm);
+  Env(NodeManager* nm, Options* opts);
```

Comment:
Since we're adding options via constructor, we can probably remove the `setOptions` method.  This method only existed due to construction order dependencies previously.

Reply:
I agree.